bindings/experimental/distrdf/python/DistRDF/Node.py

```diff
@@ -557,29 +557,6 @@ def _get_clustered_ranges(self, treename, filelist,
 
         return clustered_ranges
 
-    def _get_filelist(self, files):
-        """
-        Convert single file into list of files and expand globbing
-
-        Args:
-            files (str, list): String containing name of a single file or list
-                with several file names, both cases may contain globbing
-                characters.
-
-        Returns:
-            list: list of file names.
-        """
-        if isinstance(files, str):
-            # Expand globbing excluding remote files
-            remote_prefixes = ("root:", "http:", "https:")
-            if not files.startswith(remote_prefixes):
-                files = glob.glob(files)
-            else:
-                # Convert single file into a filelist
-                files = [files, ]
-
-        return files
```

Comment:
the removal of this method does not have anything to do with not supporting in-memory trees right? it seems it was just not needed..?

Reply:
ah no the logic has been inlined

Reply:
Indeed, the logic is now inside the `get_inputfiles` function. This is part of a series of improvements I'm bringing to distrdf offline that will appear in a PR, but since it was also involved in resolving the linked issue, I brought it up in this PR directly